definitions/tools/pvacseq_combine_variants.cwl

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env cwl-runner
+
+cwlVersion: v1.0
+class: CommandLineTool
+label: "Combine germline and somatic vcf for pVACseq phasing"
+baseCommand: ["/usr/bin/java", "-Xmx8g", "-jar", "/opt/GenomeAnalysisTK.jar", "-T", "CombineVariants"]
+requirements:
+    - class: ResourceRequirement
+      ramMin: 8000
```

Comment:
We should make sure that the resource request on these is greater than the JVM memory request!